buildroot-external/patches/occu/0093-WebUI-AllowAdminUserChanges.patch

```diff
@@ -0,0 +1,41 @@
+--- occu/WebUI/www/rega/esp/system.fn.orig
++++ occu/WebUI/www/rega/esp/system.fn
+@@ -1164,7 +1164,7 @@
+     oUser = dom.GetObject( system.GetVar("userid") );
+     if( oUser )
+     {
+-      if( !oUser.Unerasable() )
++      if( !oUser.Unerasable() || oUser.ID() == 1004 )
+       {
+         object oFavorites = dom.GetObject( ID_FAVORITES );
+         if( oFavorites )
+--- occu/WebUI/www/rega/pages/msg/userAccountConfigAdmin.htm.orig
++++ occu/WebUI/www/rega/pages/msg/userAccountConfigAdmin.htm
+@@ -25,7 +25,7 @@
+       if(userToTest.Name() == "Admin") 
+       {
+         sDisabled = "disabled";
+-        sSuperAdminDisabled = "disabled";
++        !sSuperAdminDisabled = "disabled";
+         sDeviceDisabled = "disabled";
+       }
+     }
+@@ -34,6 +34,7 @@
+ </div>
+ 
+ <div class="CLASS00700">
++  <script type="text/javascript">onPwChange = function(){};</script>

```

Comment:
A dummy functions is needed to prevent javascript errors when the paged is called for the first time where the  `onPwChange` function is not available.